.github/workflows/sonar.yaml

```
@@ -1,9 +1,7 @@
 name: SonarCloud Analysis
 
-#push: [main]
 on:
-  push:
-  pull_request:
+  push: [main]
```

actionlint (1.7.10)

[error] "push" section is sequence node but mapping node is expected

(syntax-check)